src/components/Body/index.test.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import Carousel from "./index";
+
+test("Render carousel component", () => {
+  render(<Carousel />);
+
+  const element = screen.getByText(/New Posts/);
+  expect(element).toBeInTheDocument();
+});
```

Comment:
Please add more testing as well

You can see some examples here:
https://medium.com/@fatma_2377/a-comprehensive-guide-to-unit-testing-in-react-js-with-vite-ea362d0daade

Reply:
bingung testing yg slidernya mbak

Reply:
muncul error ini mbak const API_BASE_URL = import.meta.env.VITE_API_BASE_URL;

Reply:
Please refer to the category testing that I had merged to the main branch

Reply:
I think you can update your feature branch with the latest main branch first and then adjust accordingly

Reply:
You need to have this file: [vite.config.js](https://github.com/project-zusammen/hidden-gem-frontend/blob/main/vite.config.js)

and then it will read `API_BASE_URL` that you can use anywhere, no need to use the `import.meta` anymore

Reply:
You can refer the test in [Category component](https://github.com/project-zusammen/hidden-gem-frontend/blob/main/src/components/Category/Category.spec.js)

what it does is that, it mocks the getCategory function that is called from src/api/category.js

```
jest.mock('../../api/category', () => ({
  getCategory: jest.fn(() => Promise.resolve({ data: [{ public_id: '1', name: 'Category 1' }, { public_id: '2', name: 'Category 2' }] })),
}));
```

Reply:
saya juga udah merge
saya udah ikuti arahan mbak
tetap juga gak bisa akses envnya.

`ReferenceError: API_BASE_URL is not defined`

Reply:
`.env` file nya ada?

`API_BASE_URL` di `.env` file value nya apa?

Reply:
udah ada mbak. valuenya API_BASE_URL=https://hiddengem.pythonanywhere.com/api